[2022-02-24 22:55:22,023] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: daily_fuel_data_ingestion.create_fuel_dataset manual__2022-02-24T22:55:20.271881+00:00 [queued]>
[2022-02-24 22:55:22,033] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: daily_fuel_data_ingestion.create_fuel_dataset manual__2022-02-24T22:55:20.271881+00:00 [queued]>
[2022-02-24 22:55:22,033] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-02-24 22:55:22,033] {taskinstance.py:1244} INFO - Starting attempt 1 of 2
[2022-02-24 22:55:22,034] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-02-24 22:55:22,045] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): create_fuel_dataset> on 2022-02-24 22:55:20.271881+00:00
[2022-02-24 22:55:22,055] {standard_task_runner.py:52} INFO - Started process 164 to run task
[2022-02-24 22:55:22,060] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'daily_fuel_data_ingestion', 'create_fuel_dataset', 'manual__2022-02-24T22:55:20.271881+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/ingest_daily_fuel_data_dag.py', '--cfg-path', '/tmp/tmp1y8ytvyy', '--error-file', '/tmp/tmp7e3ly2pa']
[2022-02-24 22:55:22,061] {standard_task_runner.py:77} INFO - Job 5: Subtask create_fuel_dataset
[2022-02-24 22:55:22,149] {logging_mixin.py:109} INFO - Running <TaskInstance: daily_fuel_data_ingestion.create_fuel_dataset manual__2022-02-24T22:55:20.271881+00:00 [running]> on host 036a2a858d52
[2022-02-24 22:55:22,224] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=daily_fuel_data_ingestion
AIRFLOW_CTX_TASK_ID=create_fuel_dataset
AIRFLOW_CTX_EXECUTION_DATE=2022-02-24T22:55:20.271881+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-02-24T22:55:20.271881+00:00
[2022-02-24 22:55:22,760] {warnings.py:110} WARNING - /opt/***/dags/ingest_daily_fuel_data_dag.py:77: FutureWarning: Dropping of nuisance columns in DataFrame reductions (with 'numeric_only=None') is deprecated; in a future version this will raise TypeError.  Select only valid columns before calling the reduction.
  final_df["national_average"] = final_df.mean(axis=1).round(1)

[2022-02-24 22:55:22,772] {python.py:175} INFO - Done. Returned value was: None
[2022-02-24 22:55:22,794] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=daily_fuel_data_ingestion, task_id=create_fuel_dataset, execution_date=20220224T225520, start_date=20220224T225522, end_date=20220224T225522
[2022-02-24 22:55:22,850] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-24 22:55:22,915] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
